### PR TITLE
Cli/user guide] 2. 登入 twcc cli

### PR DESCRIPTION
**Pull request by ting88510**
#249

### CODE REVIEW

File: CLI/1. User Guides/2. 登入.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: 登入 TWCC CLI | en
+tags: Guide, CLI, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/twccheader-en %}
+{%hackmd @docsharedstyle/default %}
+
+# Log in to TWCC CLI
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:10]: Sign in with TWCC CLI

File: CLI/2. Concepts/5. 虛擬磁碟/1. mk vds.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (VigaWei) [inline CLI/2. Concepts/5. 虛擬磁碟/1. mk vds.md:1]: delete this file?

File: CLI/2. Concepts/5. 虛擬磁碟/2. ls vds.md [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (VigaWei) [inline CLI/2. Concepts/5. 虛擬磁碟/2. ls vds.md:1]: delete this file?

File: CLI/1. User Guides/2. 登入.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+tags: Guide, CLI, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/twccheader-en %}
+{%hackmd @docsharedstyle/default %}
+
+# Log in to TWCC CLI
+
+After installation, please prepare **Project ID**, **API key (login certificate)**, and then log in to TWCC CLI to start.
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:12]: prepare > prepare your

log in to TWCC CLI to start. > sign in with TWCC CLI to get started.

File: CLI/1. User Guides/2. 登入.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+## Preparation
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:14]: Preparation > Prerequisites

File: CLI/1. User Guides/2. 登入.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+### 1. Project ID
+
+Sign in to [TWCC Portal](https://www.twcc.ai/), and check the **Project ID** of the using project on the homepage
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:18]: add a period, thx!

File: CLI/1. User Guides/2. 登入.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+### 2. API key
+
+- Click **the user name** in the upper right corner, and then click **API Key**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_00e143d58b5cfee093c4c95d32dd2042.png)
+
+
+
+- Click <i class="fa fa-clone" aria-hidden="true"></i> copy button to get your API key
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:33]: add a period, thx!

File: CLI/1. User Guides/2. 登入.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+Sign in to [TWCC Portal](https://www.twcc.ai/), and check the **Project ID** of the using project on the homepage
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_1a1bd7090cfd898384a84b61ef57b747.png)
+
+
+
+
+### 2. API key
+
+- Click **the user name** in the upper right corner, and then click **API Key**.
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:27]: Click*3 > Select or select

File: CLI/1. User Guides/2. 登入.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+:::danger
+{%hackmd @docsharedstyle/important-en %}
+ 
+To ensure security, please:
+- **Avoid using the master key to use services, and do not leak the master key out**.
+- Update your key regularly (delete and then create a new one).
+:::
+
+
+## Log in to TWCC CLI
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:48]: Sign in with TWCC CLI

File: CLI/1. User Guides/2. 登入.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+- Enter the following commands, and enter **TWCC API key** and **Project ID** to log in to TWCC CLI
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:50]: **TWCC API key** and **Project ID**  >

 your **TWCC API key** and **Project ID**

Comment (VigaWei) [reply CLI/1. User Guides/2. 登入.md:50]: to log in to TWCC CLI >

to sign in.

File: CLI/1. User Guides/2. 登入.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+- Language and other parameter settings
+```bash
+$ export LANG=C.UTF-8
+$ export LC_ALL=C.UTF-8
+$ export PYTHONIOENCODING=UTF-8
+```
+
+## View the user information
+
+View user information, API key, and Poject ID
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:65]: View the user information, API key and the project ID.

File: CLI/1. User Guides/2. 登入.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_357bb8bf5d9208b9e97187cbc56c58c9.png)
+
+
+:::info
+
+{%hackmd @docsharedstyle/note-en %}
+
+**How to switch to another project?**
+
+Use the following command to delete the key, and <ins>[log in to TWCC CLI](#Log-in-to-TWCC-CLI) again</ins>, then you can enter the API key of another project and use it.
```
Comment (VigaWei) [inline CLI/1. User Guides/2. 登入.md:80]: [log in to TWCC CLI](#Log-in-to-TWCC-CLI) > [Sign in with TWCC CLI](#Sign-in-with-TWCC-CLI)

Comment (VigaWei) [reply CLI/1. User Guides/2. 登入.md:80]: then you can enter the API key of another project and use it. >


then you can switch to another project by entering your API key under the project.

**Review by VigaWei — COMMENTED** [review]
Thx! Please revise the doc according to the comments.

**Comment by ting88510** [comment]
Revised, thanks!!

**Pull request merged**